docs/modules/ROOT/pages/hazelcast-platform-operator-wan-replication.adoc

```diff
@@ -14,17 +14,17 @@ In this tutorial, you'll do the following:
 
 - Deploy two Hazelcast clusters.
 
-- Create a Hazelcast map configuration.
+- Create two Hazelcast map configurations on one of the clusters.
 
-- Synchronize map data between two Hazelcast clusters.
+- Synchronize map data between the two Hazelcast clusters.
 
 == Before you Begin
 
-Before starting this tutorial, make sure that you meet the following prerequisites:
+Before starting this tutorial, make sure that you have the following:
 
-* Up and running https://kubernetes.io/[Kubernetes] cluster
-* Kubernetes command-line tool, https://kubernetes.io/docs/tasks/tools/#kubectl[kubectl]
-* Deployed xref:operator:ROOT:index.adoc[Hazelcast Platform Operator]
+* A running https://kubernetes.io/[Kubernetes] cluster
+* The Kubernetes command-line tool, https://kubernetes.io/docs/tasks/tools/#kubectl[kubectl]
+* A deployed xref:operator:ROOT:index.adoc[Hazelcast Platform Operator]
 
 == Step 1. Start the Hazelcast Cluster
 
```

Comment:
In this step, maybe you could just have the instruction below and remove the instruction on line 31.
```suggestion
. Create a secret with your link:http://trialrequest.hazelcast.com/[Hazelcast Enterprise License].
```